src/DynamoCore/Graph/Workspaces/WorkspaceModel.cs

```diff
@@ -1491,5 +1557,100 @@ protected void Log(string msg, WarningLevel severity)
+                    nodeModel.Y = nodeInfo.Y;
+                    nodeModel.Name = nodeInfo.Name;
+
+                    // Note: These parameters are not directly accessible due to undo/redo considerations
+                    //       which should not be used during deserialization (see "ArgumentLacing" for details)
+                    nodeModel.UpdateValue(new UpdateValueParams("IsVisible", nodeInfo.IsVisible.ToString()));
+                    nodeModel.UpdateValue(new UpdateValueParams("IsUpstreamVisible", nodeInfo.IsUpstreamVisible.ToString()));
+                }
+            }
+
+            foreach (ExtraNoteViewInfo noteInfo in viewInfo.Notes)
+            {
+                var guidValue = IdToGuidConverter(noteInfo.Id);
+
+                // TODO: Figure out if ZIndex needs to be set here as well
+                var noteModel = new NoteModel(noteInfo.X, noteInfo.Y, noteInfo.Text, guidValue);
+                this.AddNote(noteModel);
+            }
+
+            foreach (ExtraAnnotationViewInfo annotationInfo in viewInfo.Annotations)
+            {
+                // TODO: Determine where to set the ID for annotations
+                //Guid guidValue = IdToGuidConverter(annotationInfo.Id);
+
+                // Create a collection of nodes in the given annotation
+                var nodes = new List<NodeModel>();
+                foreach (string nodeId in annotationInfo.Nodes)
+                {
+                  var guidValue = IdToGuidConverter(nodeId);
+                  if (guidValue == null)
+                    continue;
+
+                  // NOTE: Some nodes may be annotations and not be found here
+                  var nodeModel = Nodes.FirstOrDefault(node => node.GUID == guidValue);
+                  if (nodeModel == null)
+                    continue;
+
+                  nodes.Add(nodeModel);
+                }
+
+                // Create a collection of notes in the given annotation
+                var notes = new List<NoteModel>();
+                foreach (string noteId in annotationInfo.Nodes)
+                {
+                  var guidValue = IdToGuidConverter(noteId);
+                  if (guidValue == null)
+                    continue;
+
+                  // NOTE: Some nodes may not be annotations and not be found here
+                  var noteModel = Notes.FirstOrDefault(note => note.GUID == guidValue);
+                  if (noteModel == null)
+                    continue;
+
+                  notes.Add(noteModel);
+                }
+
+                var annotationModel = new AnnotationModel(nodes, notes);
+                annotationModel.AnnotationText = annotationInfo.Title;
+                annotationModel.FontSize = annotationInfo.FontSize;
+                annotationModel.Background = annotationInfo.Background;
+                this.AddNewAnnotation(annotationModel);
+            }
+
+            // TODO: These items are not in the extra view info
+            //Name = info.Name;
```

Comment:
are these missing from the serialized json or just not implemented yet in `ExtraViewInfo`?